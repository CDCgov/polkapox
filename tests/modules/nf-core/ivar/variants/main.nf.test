nextflow_process {

    name "Test Process IVAR_VARIANTS"
    script "modules/nf-core/ivar/variants/main.nf"
    process "IVAR_VARIANTS"

    test("Ivar Variants Test") {

        when {
            params {
            }
            process {
                """
                input[0] = Channel.of([
                        [ id: 'test_sub'],
                        [
                            file("${projectDir}/tests/nf-test-data/test_sub.bam", checkIfExists: true)
                        ]
                    ])
                input[1] = true
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
