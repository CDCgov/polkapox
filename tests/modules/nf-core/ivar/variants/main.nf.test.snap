{
    "Ivar Variants Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "tsv": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-07T16:01:06.448112282"
    }
}